websites/D/daily.dev/metadata.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "937757295453044806",
+		"name": "atsukoro"
+	},
+	"service": "daily.dev",
+	"description": {
+		"en": "The best developer social platform with latest technology-related news"
```

It may be better to use the official description/text
```suggestion
		"en": "We know how hard it is to be a developer. It doesn't have to be. Personalized news feed, dev community and search, much better than what's out there. Maybe ;)"
```